### ISSUE TITLE
1.9.5 release notes.

### ISSUE THREAD

**Issue by illume** [pygame/pygame#561]
Need to gather together the release notes like in the previous release.

- [x] should be in markdown format(for the github release notes), which can be converted to html (for the website).
- [x] make a list of people to thank for the credits section.
- [x] link to all the PRs.
- [x] make a summary including important or interesting features/fixes.

Full pygame 1.9.5 release checklist here: https://github.com/pygame/pygame/issues/495

**Comment by illume** [comment]
I started this some months before, below is where I got up to then. But there's quite a few PRs and other people to thank.

```
<p align="center"><a href="https://www.pygame.org/wiki/about"><img src="https://www.pygame.org/images/logo.png" alt="pygame 1.9.4" width="300px"/></a></p>


## TLDR; Some highlights.


## Thanks

A very special thanks to the people who have volunteered commits to pygame since the last release. In alphabetical order...

(@alphaCTzo7G) | (@augusto2112)  (@e1000) | Marius Gedminas (@mgedmin) | René Dudfield (@illume) | 

## More details.

https://github.com/pygame/pygame/pull/488 Added unit test 'test_zero_surface_transform' for issue 411
https://github.com/pygame/pygame/pull/486 Moved jquery.plugin.docscomments.js into pygameweb.
https://github.com/pygame/pygame/pull/485 Repo cleanup. 8 things in top level. Rest in buildconfig/. Old stuff removed.
https://github.com/pygame/pygame/pull/484 Added mac sdl2 support. Changed config.py -sdl2
https://github.com/pygame/pygame/pull/483 Support older mac core duo 2 cpus
https://github.com/pygame/pygame/pull/482  The giant SDL2_patches merging PR.
https://github.com/pygame/pygame/pull/476 Do not try to add portmidi on freebsd. It does not exist.
https://github.com/pygame/pygame/pull/475 Resurrect Python 3.4 builds on Appveyor
```

---

<p align="center"><a href="https://www.pygame.org/wiki/about"><img src="https://www.pygame.org/images/logo.png" alt="pygame 1.9.4" width="300px"/></a></p>

## TLDR; Some highlights.


## Thanks

A very special thanks to the people who have volunteered commits to pygame since the last release. In alphabetical order...

(@alphaCTzo7G) | (@augusto2112) | (@e1000) | Marius Gedminas (@mgedmin) | René Dudfield (@illume) | 

## More details.

https://github.com/pygame/pygame/pull/488 Added unit test 'test_zero_surface_transform' for issue 411
https://github.com/pygame/pygame/pull/486 Moved jquery.plugin.docscomments.js into pygameweb.
https://github.com/pygame/pygame/pull/485 Repo cleanup. 8 things in top level. Rest in buildconfig/. Old stuff removed.
https://github.com/pygame/pygame/pull/484 Added mac sdl2 support. Changed config.py -sdl2
https://github.com/pygame/pygame/pull/483 Support older mac core duo 2 cpus
https://github.com/pygame/pygame/pull/482  The giant SDL2_patches merging PR.
https://github.com/pygame/pygame/pull/476 Do not try to add portmidi on freebsd. It does not exist.
https://github.com/pygame/pygame/pull/475 Resurrect Python 3.4 builds on Appveyor

---

**Comment by dlon** [comment]
Maybe some of these could be added:

#922 #881 #879 Fix zero-sized mask segfaults
#920 Fix the SystemError when no type specific clipboard content exists.
#917 Update scrap tests
#914 Update freetype tests
More test cleanups.
#911 #893 #889 #876 #874 #844 #840 #831 #819 Improved mask tests
#900 Add sprite tests
#891 Add missing Vector2.magnitude()
#873 Fix mask index out of bounds segmentation fault
#806 Mask constructor 'fill' argument
#805 Event functions: pump parameter and keyword arguments
#803 Add customized repr()/str() output for Mask objects
#802 Fix aaline()/aalines() not drawing on a surface's border
#760 #707 Load images on multiple threads properly
#783 Fix overlap_mask() making incorrect mask
#751 #749 #748 #746 #745 Added get_init() to all modules with an init() function
#725 Allow camera module to be used on Windows
#722 API version macros
#698 Documentation improvements
#670 Converting between ANSI notes and MIDI notes or frequencies
#663 Compile for PyPy 3 and PyPy fixes
#656 #649 Handle Unicode objects and paths properly
#630 Fix freetype rotation rendering
#596 aaline cleanup
#582 draw.aaline: blend correctly
#577 #573 Fixed FreeType memory leaks
#571 Fix surface.blits() bugs

**Comment by illume** [comment]
The new contributors need to be added from pr 571 onwards.

Also, there's been a lot of changes since I wrote the release summary.

**Comment by dlon** [comment]
I added the new contributors
